lib/BudgetButton/button.dart

```diff
@@ -1,21 +1,21 @@
 import 'package:flutter/material.dart';
-import 'package:budgie/SecondPage.dart';
+import 'package:budgie/BudgetButton/SecondPage.dart';
 
-/* void main() {
+void main() {
   runApp(MaterialApp(
     title: 'route',
-    home: _addButton(),
+    home: AddButton(),
   ));
-} */
+}
 
-class addButton extends StatefulWidget {
-  const addButton({Key? key}) : super(key: key);
+class AddButton extends StatefulWidget {
```

Comment:
What is the purpose of this button?

Reply:
I'm not too sure, but Manpreet edited my code previously to make it a little bit better. We can talk about this in our next meeting.

Reply:
My bad--I think I was trying to match up the button with the rest of the home page elements I was working with and mistakenly made it stateful.